src/ui_parts/display/menu/zoom_menu/zoom_menu.gd

```diff
@@ -0,0 +1,54 @@
+extends HBoxContainer
+
+
+const MIN_ZOOM = 0.125
+
+const MAX_ZOOM = 64.0
+
+signal zoom_changed(zoom_level: float)
+
+signal zoom_reseted()
+
```

Comment:
Add onready variables for the buttons in this file too.

Code style nitpicks:

- No whitespace between contextually related lines (like min zoom and max zoom, or the signals).
- One whitespace between contextually similar functions, two otherwise.
- zoom_in() and zoom_out() should be next to each other.